src/ssl/mod.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(unused)]
```

Comment:
whats going on here, is this fixing some existing warning?

Reply:
done by change it to dead_code and add comment